docs/.vuepress/theme/styles/theme.styl

```diff
@@ -133,6 +133,10 @@ ul
   li
     padding-left 0.5em
 
+    &.highlight
```

Comment:
Not sure if this is what caused it but looks like the light theme highlight now displays in dark mode:
<img width="341" alt="Image 2020-02-28 at 12 29 37 PM" src="https://user-images.githubusercontent.com/8097623/75548842-140ee900-5a26-11ea-950d-07d66689102d.png">

Reply:
Resolved